lib/koans/03_atoms.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Atoms do
+  use Koans
+
+  koan "Atoms are sort of like strings" do
+    adam = :human
+    assert adam == ___
+  end
+
+  koan "Strings can be converted to atoms, and vice versa" do
+    assert String.to_atom("atomized") == ___
+    assert Atom.to_string(:stringified) == ___
+  end
+
+  koan "Atoms are often used as keys, because they're faster than strings" do
+    map = %{name: "Jay"}
+    list = [name: "Jay"]
+
+    assert map[:name] == ___
+    assert list[:name] == ___
+  end
+
+  #koan "Only atom keys may be accessed with dot syntax" do
+  #  map = %{name: "Jay"}
+  #  assert map.name == ___
+
+  #  map = %{"name" => "Jay"}
+  #  assert_raise KeyError, fn -> ___ end
+  #  assert map["name"] == ___
+  #end
+
+  #koan "Dot syntax is stricter than access with brackets" do
+  #  map = %{name: "Jay"}
+
+  #  assert map[:age] == ___
+  #  assert_raise KeyError, fn -> ___ end
+  #end
+
+  koan "It is surprising to find out that booleans are atoms" do
+    assert is_atom(true) == ___
+    assert is_atom(false) == ___
+    assert :true == ___
+    assert :false == ___
+  end
+
+  koan "Modules are also atoms" do
+    assert is_atom(String) == ___
+    assert :"Elixir.String" == ___
```

Comment:
I suspect the intention here is to fill it out like this: `assert :"Elixir.String" == String`. I doubt that people will actually do that, though, because it's not obvious behaviour in Elixir and it's much easier to make it pass with `assert :"Elixir.String" == :"Elixir.String"`.

Thoughts?